README.md

```diff
@@ -9,29 +9,33 @@
 [![Twitter Follow](https://img.shields.io/twitter/follow/vr_toolkit.svg?style=flat&label=twitter)](https://twitter.com/VR_Toolkit)
 [![Waffle](https://img.shields.io/badge/project-backlog-78bdf2.svg)](https://waffle.io/ExtendRealityLtd/VRTK.Unity.Core)
 
-## 3<sup>rd</sup> Party SDK Support
-
-* [VRTK.Unity.SteamVR]
-* [VRTK.Unity.OculusUtilities]
-
 ## Introduction
 
 VRTK.Unity.Core is a core collection of design patterns for [Unity3d] that can be beneficial in VR development.
 
+  > VRTK.Unity.Core requires Unity3d 2018.3 (or above).
+
 ## Getting Started
 
 ### Setting up a project
 
-* Using Unity3d 2018.1 or above create a new project using the 3D Template or open an existing project.
-* Update the project `Scripting Runtime Version` to `.NET 4.x Equivalent`.
-  * In Unity3d select `Main Menu -> Edit -> Project Settings -> Player` to open the `Player Settings` inspector.
-  * In the `PlayerSettings` inspector panel expand `Other Settings`.
-  * Change the `Scripting Runtime Version` to `.NET 4.x Equivalent`.
-  * Unity3d will now restart in the supported scripting runtime.
-* If the project supports Virtual Reality then ensure `Virtual Reality Supported` is checked.
-  * In Unity3d select `Main Menu -> Edit -> Project Settings -> Player` to open the `Player Settings` inspector.
-  * In the `PlayerSettings` inspector panel expand `XR Settings`.
-  * In `XR Settings` ensure the `Virtual Reality Supported` option is checked.
+* Using Unity3d 2018.3 (or above) create a new project using the 3D Template or open an existing project.
+* If the project supports Virtual Reality:
+  * Ensure `Virtual Reality Supported` is checked.
+    * In Unity3d select `Main Menu -> Edit -> Project Settings` to open the `Project Settings` inspector.
+    * Select `Player` from the left hand menu in the `Project Settings` inspector.
+    * In the `Player` inspector panel expand `XR Settings`.
+    * In `XR Settings` ensure the `Virtual Reality Supported` option is checked.
+  * Install the appropriate support package.
```

Comment:
```suggestion
  * Ensure the appropriate support package is installed.
```
Unity _should_ be adding the package automatically once enabled, thus these instructions just ensure that, they don't necessarily add the package as they may already exist.